nodejs/aws-infra/ec2/cidr.ts

```diff
@@ -126,15 +126,15 @@ function getIPv4AddressValue(address: string): number {
     function getOctet(piece: string): number {
         const val = parseInt(piece, 10);
         if (val < 0 || val >= 256) {
-            throw new Error(`Each part of the ip address must be between 0 and 255. Got "${val}"`);
+            throw new Error(`Each part of the ip address must be between 0 and 255. Parsed "${val}" from "${piece} in ${address}"`);
         }
 
         return val;
     }
 }
 
 export function getIPv4Address(value: number): string {
```

Comment:
Not sure if you've got a Node test harness set up, but it would be awesome if we could turn this into a normal Node unit test 😄

Reply:
Yeah.  we could def make these into unit tests as the cidr object is just pure js.  let me file issue to track that.